lightning/Cargo.toml

```diff
@@ -28,6 +28,7 @@ unstable = []
 
 [dependencies]
 bitcoin = "0.26"
+zbase32 = "0.1.2"
```

Comment:
Hmm, I'm not a huge fan of depending on a library for something that should be 20-50 LoC, plus it looks like zbase32 doesn't meet our MSRV (hence why CI is failing). Sadly we can't just copy it and fix it since its LGPL, so we'll need to write one.

Reply:
Got it. I'll add our own implementation of zbase32.

Reply:
Added a local `zbase32` encoding based on https://github.com/andreasots/base32 in c7f405e